hide_invoice_shipping_address/views/account_move_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2024 Roger Sans <roger.sans@sygel.es>
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="account_move_form_inherit">
+            <field name="name">account.move.form.inherit</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='partner_shipping_id']" position="after">
+                    <field name="invoice_shipping_address_print" 
+                        attrs="{'invisible':[('financial_type','not in',['receivable','receivable_refund'])]}"/>
```

Comment:
Please, use ('move_type', 'not in',  ['out_invoice', 'out_refund']). financial_type is a field created by the module account_tax_balance, which might not be installed.

Reply:
Done